service/controller/catalog/resource/appcatalogsync/create.go

```diff
@@ -0,0 +1,56 @@
+package appcatalogsync
+
+import (
+	"context"
+
+	"github.com/giantswarm/apiextensions/v3/pkg/apis/application/v1alpha1"
+	"github.com/giantswarm/app/v5/pkg/key"
+	"github.com/giantswarm/microerror"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// EnsureCreated ensures appcatalog CRs are created for compatibility with catalog CRs
+func (r *Resource) EnsureCreated(ctx context.Context, obj interface{}) error {
+	if !r.uniqueApp {
+		// Return early. Only unique instance manages appcatalogentry CRs.
+		return nil
+	}
+
+	cr, err := key.ToCatalog(obj)
+	if err != nil {
+		return microerror.Mask(err)
+	}
+
+	if namespace := cr.GetNamespace(); namespace == metav1.NamespaceDefault || namespace == "giantswarm" {
```

Comment:
I think customers should not care about catalog CRs in the default namespace or giantswarm namespace.  So I would skip creating appcatalog CRs in these cases. WDYT?

Reply:
Yes, I think that makes sense.